src/msw/webview_edge.cpp

```diff
@@ -761,6 +785,19 @@ bool wxWebViewEdge::IsAccessToDevToolsEnabled() const
     return true;
 }
 
+bool wxWebViewEdge::SetUserAgent(const wxString& userAgent)
+{
+    m_impl->m_customUserAgent = userAgent;
+    // Can currently only be set before Create()
+    if (m_impl->m_webViewController)
```

Comment:
Should we assert here, i.e. use `wxCHECK_MSG( !m_impl->m_webViewController, false, "Can't be called after Create()" );`?

It seems like this could be more helpful.